### ISSUE TITLE
Github integrations for Janis

### ISSUE THREAD

**Issue by Niicck** [cityofaustin/techstack#4337]
Since we're doing manual netlify deployments with Publisher2, we don't get that nice automated github integration anymore. So on a Janis PR, we don't have an accurate sense of when/if a Janis build succeeded. And the link to that Janis site is no longer automatically populated on the issue. We should make that happen with a webhook call that gets fired from Publisher after a success or failure.

**Comment by chiaberry** [comment]
Nick did this!

**Comment by Niicck** [comment]
Nope, I didn't. A Janis PR on github doesn't know when/if a Publisher build for your Janis code actually succeeded or not. And it doesn't have a nice link to the final built Janis.
